cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -266,12 +273,14 @@ std::vector<bitmask_type> make_null_mask_vector(ValidityIterator begin, Validity
  * element in `[begin,end)` that evaluated to `true`.
  */
 template <typename ValidityIterator>
-rmm::device_buffer make_null_mask(ValidityIterator begin, ValidityIterator end)
+std::pair<rmm::device_buffer, cudf::size_type> make_null_mask(ValidityIterator begin,
+                                                              ValidityIterator end)
 {
-  auto null_mask = make_null_mask_vector(begin, end);
-  return rmm::device_buffer{null_mask.data(),
-                            null_mask.size() * sizeof(decltype(null_mask.front())),
-                            cudf::get_default_stream()};
+  auto [null_mask, null_count] = make_null_mask_vector(begin, end);
+  auto d_mask                  = rmm::device_buffer{null_mask.data(),
+                                   null_mask.size() * sizeof(decltype(null_mask.front())),
```

Comment:
No strong opinion here, but aren't we basically guaranteed that `decltype(null_mask.front())` is `bitmask_type`? Can we ever have anything else?

Reply:
That is strange. I'd not noticed it before. I think we should be using https://github.com/rapidsai/cudf/blob/a562a7e0ffbe3fcfa78488733ecb00beebeaa48b/cpp/include/cudf/null_mask.hpp#L57 here instead.